src/components/forms/NetworkDevicesForm/NetworkDeviceContent.tsx

```diff
@@ -82,6 +104,22 @@ const NetworkDeviceContent: FC<Props> = ({
         formik={formik}
         canSelectManualAcls={supportsNicDeviceAcls(network)}
       />
+      <NetworkDeviceIPAddress
+        formik={formik}
+        index={index}
+        network={network}
+        readOnly={readOnly}
+        device={device}
+        family="IPv4"
+      />
+      <NetworkDeviceIPAddress
+        formik={formik}
+        index={index}
+        network={network}
+        readOnly={readOnly}
+        device={device}
+        family="IPv6"
+      />
```

Comment:
I think this needs the `formik.values.entityType == "instance"` guard as well as the readmode above to avoid leaking this into profile configuration edit.